app/apis/ghn_client.rb

```diff
@@ -59,6 +59,7 @@ def create_order(order)
     service_response = get_service(order)
     service_id = service_response["data"][0]["service_id"]
     service_type_id = service_response["data"][0]["service_type_id"]
+    cod_amount = order.payment.status == "paid" ? 0 : (order.total_price + 20000)
```

Comment:
avoid magic number here. Set `20000` as a constant in order model

Reply:
noted and updated